[IMP] README: improve pre-commit instructions (#658)
OCA/maintainer-quality-tools: README.md

The goal of Maintainer Quality Tools (MQT) is to provide helpers to ensure the quality of Odoo addons.

> If you are a developer and are looking to run the pre-commits action locally,

Reviewer: ```suggestion
If you are a developer and are looking to run the pre-commit actions locally,
```

you don't need this repository.
Follow the [pre-commit tool](https://pre-commit.com/#install) installation instructions.

In a nutshell:

> - Install pre-commit in your system: ``pip3 install pre-commit``

Reviewer 2: ```suggestion
- Install pre-commit in your system: ``pip3 install --user pre-commit``
```

Recommending `--user` to avoid user that get a permission issue to retry with sudo and break their system.

- Install pre-commit on yor repository (must be done inside each OCA repo):
  ``pre-commit install``

> - If you want to manually run it: ``pre-commit pre-commit run --all-files``

Reviewer 2: ```suggestion
- If you want to manually run it: ``pre-commit run --all-files``
```
